docs/users.rst

```diff
@@ -44,7 +44,35 @@ Creating a build JSON
 Writing a Dockerfile
 --------------------
 
-.. _container.yaml:
+A Dockerfile is required for building container images in OSBS. It must be
+placed at the root of git repository. There can only be a single Dockerfile per
+git repository branch.
+
+Some labels are required to be defined:
+
+- ``com.redhat.component``: the value of this label is used when importing a
+  build into Koji via content generator API.
+- ``name``: value is used to define the repository name in container registry to
+  push built image. Limit this to alphanumerical values. A single ``/`` is allowed.
```

Comment:
Actually this is lowercase alphanumeric characters with the possibility to use dash as a word separator.